consensus/buildtest.go

```diff
@@ -32,7 +31,7 @@ var (
 
 	CoinbaseAugment = new(big.Int).Lsh(big.NewInt(1), 80)
```

Comment:
okay, we have two options here then:
```go
CoinbaseAugment = new(big.Int).Exp(big.NewInt(10), big.NewInt(24), nil)
```
or:
```go
CoinbaseAugment, _ = new(big.Int).SetString("1000000000000000000000000", 10) // 10^24
```
They're both pretty ugly. I don't like the extra `nil` on the end of `Exp`, but it's probably better than using a string.